test/integration/bodyDictionary.spec.ts

```diff
@@ -0,0 +1,496 @@
+import { assert } from "chai";
+import { BodyDictionaryClient } from "./generated/bodyDictionary/src/bodyDictionaryClient";
+import {
+  DictionaryGetBase64UrlResponse,
+  DictionaryGetBooleanTfftResponse,
+  DictionaryGetBooleanInvalidNullResponse,
+  DictionaryGetBooleanInvalidStringResponse,
+  DictionaryGetIntegerValidResponse,
+  DictionaryGetIntInvalidNullResponse,
+  DictionaryGetIntInvalidStringResponse,
+  DictionaryGetLongValidResponse,
+  DictionaryGetLongInvalidNullResponse,
+  DictionaryGetLongInvalidStringResponse,
+  DictionaryGetFloatValidResponse,
+  DictionaryGetFloatInvalidNullResponse,
+  DictionaryGetFloatInvalidStringResponse
+} from "./generated/bodyDictionary/src/models";
+
+describe("BodyDictionary", () => {
+  let client: BodyDictionaryClient;
+
+  beforeEach(() => {
+    client = new BodyDictionaryClient();
+  });
+
+  it("should get empty", async () => {
+    const result = await client.dictionary.getEmpty();
+    assert.lengthOf(Object.keys(result), 0);
+  });
+
+  it("should put an empty dictionary", async () => {
+    const result = await client.dictionary.putEmpty({});
+    assert.equal(result._response.status, 200);
+  });
+
+  it("should get null dictionary", async () => {
+    const result = await client.dictionary.getNull();
```

Comment:
If you were to `console.log(result)`, would it appear as though it were an empty object? That's what I was seeing with arrays (well, empty array instead of object).

Reply:
Yup it would be {}